lib/CSSStyleDeclaration.js

```diff
@@ -17,7 +17,10 @@ function parseStyles(s) {
   parser.addListener("property", function(e) {
     if (e.invalid) return; // Skip errors
     result[e.property.text] = e.value.text;
-    if (e.important) result.important[e.property.text] = e.important;
+    if (e.important) {
+      result.important = result.important || {};
```

Comment:
Instead of doing this check every time, `result` is defined above. Why not do `var result = { important: {} };`?

Also, naive question, is there a chance `e.property.text === 'important'`? In which case, L19 may be problematic.